website/docs/d/redis_cache.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_redis_cache"
+sidebar_current: "docs-azurerm-datasource-redis-cache"
+description: |-
+  Gets information about an existing Azure Redis Cache.
+
+---
+
+# Data Source: azurerm_redis_cache
+
+Use this data source to access information about an existing Redis Cache
+
+# Example Usage
+```hcl
+data "azurerm_redis_cache" "example" {
+  name                = "myrediscache"
+  resource_group_name = "redis-cache"
+}
+
+output "primary_access_key" {
+  value = "${data.azurerm_redis_cache.example.primary_access_key}"
+}
+
+output "hostname" {
+  value = "${data.azurerm_redis_cache.example.hostname}"
+}
+
+```
+
+## Argument Reference
+
+* `name` - (Required) Specifies the name of the Redis cache
+* `resource_group_name` - (Required) Specifies the name of the resource group the Redis cache instance is located in 
+
+## Attribute Reference
+
+* `id` - The Route ID.
+
+* `hostname` - The Hostname of the Redis Instance
+
+* `ssl_port` - The SSL Port of the Redis Instance
+
+* `port` - The non-SSL Port of the Redis Instance
+
+* `primary_access_key` - The Primary Access Key for the Redis Instance
+
+* `secondary_access_key` - The Secondary Access Key for the Redis Instance
+
+* `redis_configuration` - A `redis_configuration` block as defined below.
+
+---
+
+A `redis_configuration` block exports the following:
+
+* `enable_authentication` - Is the Redis instance will be accessible without authentication?
+
+* `maxmemory_reserved` - The value in megabytes reserved for non-cache usage e.g. failover
```

Comment:
could we document the remaining blocks/fields in here? generally speaking we copy these from the resource & update the tone e.g. `Is X enabled`, `The value of` rather than `Specifies the value for X`

Reply:
I took the original export list from the Attributes section in the resource doc, which only currently lists `maxclients`. Should I update this also, or address separately?

Reply:
I've not updated the r/ docs in this PR, but have completed the schema for the d/ doc.